### DISCUSSION TITLE
能不能在设置中留个勾子，允许添加disqus或自建的第三方评论代码。

### DISCUSSION THREAD

**Discussion by skyue** [usememos/memos#1044] [Ideas]
我把memos当微博用，挺想要个评论功能。

考虑到memos定位是简洁的笔记应用，直接内置评论挺奇怪的。

但如果修改主题，嵌入第三方评论系统，是不是挺好的。

**Comment by sprite82y** [comment]
设置里有个自定义脚本功能，应该可以用js实现你要的功能
